debian/rules

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/make -f
+# -*- makefile -*-
+
+# Uncomment this to turn on verbose mode.
+#export DH_VERBOSE=1
+
+VERSION=$(shell dpkg-parsechangelog -SVersion)
+
+# include /usr/share/dpkg/default.mk
+
+%:
+	dh $@ --parallel --fail-missing --with systemd
+
+override_dh_auto_build:
+	cat data/anbox.service.in | sed s:@SNAP_MOUNT_DIR@:/snap/bin:g > data/anbox.service
+	cat data/anbox.conf.in | sed s:@SNAP_MOUNT_DIR@:/snap/bin:g > data/anbox.conf
+
+override_dh_install:
+	VERSION=$(shell dpkg-parsechangelog -SVersion)
+	install -d $(CURDIR)/debian/tmp/usr/src
```

Comment:
unless you want to make this buildable on HURD, you should use DEB_DESTDIR (should point to debian/tmp/) instead of $(CURDIR)/debian/tmp/ (equally for the rest below)